sanic/server.py

```diff
@@ -955,14 +955,14 @@ def serve(
 def _build_protocol_kwargs(
     protocol: Type[HttpProtocol], config: Config
 ) -> dict:
-    if hasattr(protocol, "websocket_timeout"):
+    if (dir(protocol).__contains__("websocket_handshake")):
         return {
-            "max_size": config.WEBSOCKET_MAX_SIZE,
-            "max_queue": config.WEBSOCKET_MAX_QUEUE,
-            "read_limit": config.WEBSOCKET_READ_LIMIT,
-            "write_limit": config.WEBSOCKET_WRITE_LIMIT,
-            "ping_timeout": config.WEBSOCKET_PING_TIMEOUT,
-            "ping_interval": config.WEBSOCKET_PING_INTERVAL,
+            "websocket_max_size": config.WEBSOCKET_MAX_SIZE,
+            "websocket_max_queue": config.WEBSOCKET_MAX_QUEUE,
```

Comment:
Why are these being renamed `websocket_`?

Reply:
Left this change in, but I updated the tests so that the change was carried through consistently.